### PR TITLE
Lee/fix login

### PR DESCRIPTION
**Pull request by Hyukjoo-Lee**
## Description

- I have resolved a login issue where the page would continue loading even with invalid user information. 

## Solution
- By utilizing the 'response.ok' value, I have checked if user information is correct or not. 
- If the information is incorrect, a toast message is displayed, and the form is not submitted by setting 'setSubmitForm: false'

## Screen
Screen Shot 2023-08-05 at 3 04 15 AM

### CODE REVIEW

File: invitation-frontend/package.json [hunk 1]
```diff
@@ -1,75 +1,76 @@
...
+    "dotenv": "^16.0.3",
+    "react": "^18.2.0",
+    "react-color": "^2.19.3",
+    "react-datepicker": "^4.8.0",
+    "react-dom": "^18.2.0",
+    "react-icons": "^4.8.0",
+    "react-responsive-carousel": "^3.2.23",
+    "react-router-dom": "^6.4.2",
+    "react-swiper": "^0.1.5",
+    "react-toastify": "^9.1.3",
```
Comment (HanKyungSung) [inline invitation-frontend/package.json:30]: Is there a specific reason why you installed a new package?

Have you tried to look at this? [https://mantine.dev/core/notification/](https://mantine.dev/core/notification/)?

**Comment by Hyukjoo-Lee** [comment]
I have employed Mantine in place of a new library. 
Could you please review and confirm if we can proceed with merging?

**Comment by HanKyungSung** [comment]
image

As the screenshot above, there are a few things we need to adjust.

- The notification box won't close even clicking the x button.
- The email and password box border should turn red to indicate either box is wrong.
  > Make sure to use the Mantine component props. I believe it is an `error` prop but double-check before using it.

**Comment by Hyukjoo-Lee** [comment]
### Description
- I have fixed an issue with the close button for the Notification component.
- I have implemented error indication for incorrect login inputs.

### Solution
- To close a Notification I added 'setNotificationVisible(true)' 
- Modified specific HTTP response status codes to identify incorrect inputs. 
response.status === 404, indicated an incorrect email
response.status === 403, indicated an incorrect password
- To indicate which input field is incorrect, I modified the handleShowNotification function, passing values (error messages and fields (either 'email' or 'password')

### Screen
Screen Shot 2023-08-15 at 2 25 59 AM

Screen Shot 2023-08-15 at 2 27 09 AM

**Review by HanKyungSung — APPROVED** [review]

**Pull request merged**